ports/stm/boards/nucleo_f446re/mpconfigboard.mk

```diff
@@ -0,0 +1,35 @@
+USB_VID = 0x239A
+USB_PID = 0x809F
```

Comment:
How did you select these IDs? It appears that this is allocated by Adafruit for a different product under the Arduino programming environment, so it may not be appropriate for this board.

Reply:
I have looked at the other boards in the stm port. All of them had the `USB_VID=0x239A` (Adafruit) and for USB_PID I have picked up one unused ( I taught :) )
![image](https://user-images.githubusercontent.com/3575408/192338251-f9c455c7-3966-413c-bfed-96e8994f9a2b.png)

Maybe it should be the VID from ST since it's a development board from them ?

Reply:
We used to give out Adafruit PID's (we have an internal master list), but we stopped doing that a while ago for various reasons. Now if you need an independent VID/PID, we suggest you go to https://github.com/pidcodes/pidcodes.github.com if you cannot get one from the manufacturer. There is some discussion of ST sublicensing PIDs here: https://www.google.com/search?q=st+get+usb+pid+site:community.st.com, but I don't see it as a simple process. Does the board identify in any way with an ST VID now?

Reply:
pid.codes on second thought may not be the best choice, since it is directed to open-source hardware. ST is the best source. Could you do some more research on that? If it seems like a dead-end I will consult internally about assigning an Adafruit one.

Reply:
I see you just pushed an ST PID. Where did you get the PID from?

Reply:
@jepler @dhalbert I replaced the USB VID and PID with the ones generated by STM32CubeIDE, and it seems to work.
![image](https://user-images.githubusercontent.com/3575408/192347225-32ea0770-4019-4ca4-a4c4-1a37171c6eb8.png)

Reply:
Since it's a development board from ST, then VID can be the one from ST. 
The PID should be the one that ST uses for a USB device that exposes itself as CDC (communication device class ) + MSC (mass storage class), but I can't figure out how to generate one.
The PID I used is the MSC generated by the STM32CubeIDE, and the board with my CircuitPython port is recognized on both my linux, and windows machines, so it's seems to work.

Reply:
This seems fine to me, especially for a dev board. Thanks for finding out about the PIDs.